lib/src/model/documentation_comment.dart

```diff
@@ -707,4 +716,145 @@ mixin DocumentationComment
       }
     });
   }
+
+  /// Returns the documentation for this literal element unless
+  /// [config.dropTextFrom] indicates it should not be returned.  Macro
+  /// definitions are stripped, but macros themselves are not injected.  This
+  /// is a two stage process to avoid ordering problems.
+  String _documentationLocal;
+
+  String get documentationLocal =>
+      _documentationLocal ??= _buildDocumentationLocal();
+
+  /// Unconditionally precache local documentation.
+  ///
+  /// Use only in factory for [PackageGraph].
+  Future<void> precacheLocalDocs() async {
+    _documentationLocal = await _buildDocumentationBase();
+  }
+
+  bool _needsPrecache;
+  bool get needsPrecache => _needsPrecache ??=
+      _needsPrecacheRegExp.hasMatch(documentationComment ?? '');
+
+  String _rawDocs;
+
+  String _buildDocumentationLocal() => _buildDocumentationBaseSync();
+
+  /// Override this to add more features to the documentation builder in a
+  /// subclass.
+  String buildDocumentationAddition(String docs) => docs ??= '';
```

Comment:
`??=`? For a parameter? Should it just be `??`?

Reply:
Oops, fixed.